### DISCUSSION TITLE
Backpack DevTools - Next Update

### DISCUSSION THREAD

**Discussion by david2701** [Laravel-Backpack/community-forum#863] [Q&A (Help)]
Hello, how are you?

I'm a newbie in the world of Laravel and Backpad.

First congratulate you for excellent work that is being done, the reason for my question is to know, when is planned the new version of Backpack DevTools with functions of Generate custom admin panel components.

Thank you very much

**Comment by jcastroa87** [comment]
Hello @david2701,

Thank you for your kind words! I'm glad to hear that you're enjoying working with Backpack.

Regarding your question about the new version of Backpack DevTools with functions to generate custom admin panel components, I'd be happy to provide more information. Could you please clarify what you mean by "custom admin panel components"? Are you referring to specific features or functionalities that you'd like to see added to DevTools for generating custom components within the admin panel? In the next version, we are planning new features like live components :)

Any additional details you can provide will help me better understand your request and give you a more accurate response.

Looking forward to hearing from you.

Cheers.

**Comment by david2701** [comment]
Thank you very much for the prompt and clear answer.

My question is related to this Featured that you are currently working on.

Attached Image

Thank you very much



<img width="765" alt="Capture d’écran 2024-03-03 à 11 16 18" src="https://github.com/Laravel-Backpack/community-forum/assets/6515966/1fc0c092-5d82-40b8-8c8d-96917a71b1bb">

**Comment by jcastroa87** [comment]
Hi, @david2701, many of these features are currently active in the latest version, I think we need to update our docs; let me know if you miss one of them.

Cheers.

**Comment by jcastroa87** [comment]
Because lack of activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
